### PR TITLE
Add return annotations, some type-hints, and reformat some code.

### PR DESCRIPTION
**Pull request by ToxicKidz**

### CODE REVIEW

File: cogs/moderation/cases.py [hunk 1]
```diff
@@ -13,12 +15,12 @@ def __init__(self, bot: Bot):
         self.bot = bot
 
     @staticmethod
-    def chunks(lst, n):
+    def chunks(lst: list, n: int) -> Generator[list, None, None]:
```
Comment (anand2312) [inline cogs/moderation/cases.py]: Unrelated to the PR, but why didn't we just use `more_itertools.chunked`?

File: cogs/moderation/moderation.py [hunk 2]
```diff
@@ -21,7 +20,7 @@ def __init__(self, bot):
     @role_hierarchy()
     async def _kick(
         self, ctx: Context, member: discord.Member, *, reason="No Reason Provided"
-    ):
+    ) -> None:
```
Comment (anand2312) [inline cogs/moderation/moderation.py:27]: Should we be type hinting the `reason` argument as `str` as well?

File: cogs/moderation/moderation.py [hunk 3]
```diff
@@ -66,7 +65,7 @@ async def _kick(
     @commands.has_permissions(ban_members=True)
     async def _ban(
         self, ctx: Context, member: discord.Member, *, reason="No Reason Provided"
-    ):
+    ) -> None:
```
Comment (anand2312) [inline cogs/moderation/moderation.py:76]: Same here, and also `permanently`  has been misspelled in the docstring here.

File: cogs/moderation/moderation.py [hunk 4]
```diff
@@ -118,9 +117,8 @@ async def tempban(
         length: CustomTimeConverter,
         *,
         reason="No Reason Provided",
-    ):
-
-        # TODO: add comments and docstrings
+    ) -> None:
```
Comment (anand2312) [inline cogs/moderation/moderation.py:128]: Why's there a `*` here btw? We declare arguments to be keyword-only in function definitions, not function calls.

File: internal/context.py [hunk 5]
```diff
@@ -18,7 +18,7 @@ async def guild_config(self) -> dict:
             return json.loads(guild["config"])
         return {}
 
-    async def update_guild_config(self, **kwargs):
+    async def update_guild_config(self, **kwargs) -> None:
```
Comment (anand2312) [inline internal/context.py]: Seeing how we've type-hinted *args in some other places, I think we should type hint **kwargs as well to maintain uniformity.

File: utilities/database.py [hunk 6]
```diff
@@ -17,50 +19,52 @@ async def setup(self):
             password=getenv("DB_PASS", "password"),
         )
 
-    async def execute(self, query: str, *args):
+    async def execute(self, query: str, *args) -> None:
```
Comment (anand2312) [inline utilities/database.py]: Type hint *args here as we've done it in some other places in the project

Comment (anand2312) [reply utilities/database.py]: Actually just realized that they could be anything 🤔 maybe typing.Any

File: utilities/decorators.py [hunk 7]
```diff
@@ -2,16 +2,16 @@
 from functools import partial, wraps
 
 
-def run_in_executor(loop=None):
+def run_in_executor(executor=None):
```
Comment (anand2312) [inline utilities/decorators.py]: Why aren't there any annotations here? (and in the inner functions)

File: utilities/help.py [hunk 8]
```diff
@@ -106,10 +108,10 @@ def make_pages(cls, embed: discord.Embed, max_embeds: int, **options):
 class Help(HelpCommand):
     """Shows help for a command, group, or cog."""
 
-    def get_command_signature(self, command: commands.Command):
+    def get_command_signature(self, command: commands.Command) -> str:
         return f"{self.clean_prefix}{command.qualified_name} {command.signature}"
 
-    async def send_bot_help(self, mapping):
+    async def send_bot_help(self, mapping) -> None:
```
Comment (anand2312) [inline utilities/help.py]: What's the type of `mapping`?

File: utilities/help.py [hunk 9]
```diff
@@ -183,7 +185,7 @@ async def send_group_help(self, group: commands.Group):
         )
         await self.context.send(embed=embed)
 
-    async def send_error_message(self, error):
+    async def send_error_message(self, error: str) -> None:
```
Comment (anand2312) [inline utilities/help.py:190]: How does this work? This function only gets a string of the error, and not the error instance itself?

**Review by anand2312 — COMMENTED** [review]
Commented a bunch of things that aren't from your PR, and a few places where I think you've missed some annotations

File: cogs/moderation/moderation.py (hunk 4 above)
Comment (ToxicKidz) [reply cogs/moderation/moderation.py:128]: This is a function definition?

File: internal/context.py (hunk 5 above)
Comment (ToxicKidz) [reply internal/context.py]: They can be any type. You want me to type hint to object?

File: utilities/help.py (hunk 8 above)
Comment (ToxicKidz) [reply utilities/help.py]: Dict

File: cogs/moderation/moderation.py (hunk 4 above)
Comment (anand2312) [reply cogs/moderation/moderation.py:128]: My bad github hiding lines in between was too much for my smol brain to handle

File: internal/context.py (hunk 5 above)
Comment (anand2312) [reply internal/context.py]: typing.Any I guess

File: utilities/help.py (hunk 9 above)
Comment (ToxicKidz) [reply utilities/help.py:190]: According to the documentation, yes.

**Comment by ToxicKidz** [comment]
Pushed

**Pull request merged**